docs/run/start/existing_node.md

```diff
@@ -0,0 +1,8 @@
+---
+sidebar_position: 6
+description: Create a DV using an existing beacon node.
+---
+
+# Use an existing beacon node
+
+You may have an existing beacon node and wish to use it to run as part of a DV cluster.
```

Comment:
Is this page supposed to be... only 1 sentence?

Reply:
Same as previous comment, this will be a brand new page, instead of having this information where it is currently at the bottom of the group quickstart as a selector tab.

Reply:
To do.

Reply:
Yeah this is another placeholder page. Need to add more content.